### PR TITLE
Bug 1802600: Add unpause VM tests

### PR DESCRIPTION
**Pull request by pcbailey**
This PR adds tests for the unpause VM functionality added in https://github.com/openshift/console/pull/3985.

### CODE REVIEW

File: frontend/packages/kubevirt-plugin/integration-tests/tests/vm.actions.scenario.ts [hunk 1]
```diff
@@ -71,6 +71,15 @@ describe('Test VM actions', () => {
       VM_ACTIONS_TIMEOUT_SECS,
     );
 
+    it(
+      'Unpauses VM',
+      async () => {
+        pauseVM(vmName, testName);
```
Comment (rhrazdil) [inline frontend/packages/kubevirt-plugin/integration-tests/tests/vm.actions.scenario.ts]: Is the pause action immediate? Shouldn't we wait for Paused status?

File: frontend/packages/kubevirt-plugin/integration-tests/views/editStatusView.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+import { $, by } from 'protractor';
+
+export const unpauseVMDialog = $(`[aria-label="Edit pause state"]`);
+export const saveButton = unpauseVMDialog.element(by.buttonText('Unpause'));
```
Comment (rhrazdil) [inline frontend/packages/kubevirt-plugin/integration-tests/views/editStatusView.ts]: Maybe better to call it unpauseButton?

File: frontend/packages/kubevirt-plugin/integration-tests/tests/vm.overview.scenario.ts [hunk 3]
```diff
@@ -123,4 +123,12 @@ describe('Test VM overview', () => {
       );
     });
   });
+
+  it('Check unpause VM when VM is running', async () => {
+    await vm.action(VM_ACTION.Start);
```
Comment (rhrazdil) [inline frontend/packages/kubevirt-plugin/integration-tests/tests/vm.overview.scenario.ts]: I think this deserves a docstring. I don't understand why opening a modal window unpauses the VM

**Comment by rhrazdil** [comment]
Verified the tests, I'm okay with current implementation.
/lgtm

**Comment by spadgett** [comment]
/hold
for test fix #4411

**Comment by jelkosz** [comment]
lets see if the cherry pick bot took the latest version:
/cherry-pick release-4.4

**Comment by jelkosz** [comment]
it did :(
@pcbailey please backport manually

**Pull request merged**